### ISSUE TITLE
Bugs on LW/HD/VR Versions in build 1.4.5.945 and one request

### ISSUE THREAD

**Issue by Nyexxi** [alloystorm/dvvr#271]
### Everything tested in a clean installation with no custom configs.

**Speciphic for LW Desktop and VR Mode**

1. **SKIN MAP**: The implemented skin map in v.1.4.2 is not working right. Under the skin shader option, detail bump create horizontal strip lines around the model, the only solution is to turn it off, loosing this feature for the skin.
2. **GLOW EFFECT**: The Glow effect for materials seems to work but not strong enough like in oldest versions making some materials that use darker colors, like purple or deep blue, to not even glow. A possible solution could be to rise the limits in the setting bar for this adjustment, not sure about this.
3. **GRAPHIC OPTIONS**: In LW VR mode, changing any graphic option in game freeze the whole menu with the only option to close the program and relaunch the program every time you change something while in VR, not even sure if the options changed are saved.

**Common bugs for LW/HD/VR Modes**

1. **BEATS VISUALIZER**: Radius size, marker size and line width options not showing changes in game until relaunching the program.
2. **VR MODE**: Show FPS option not working, it doesn't show the FPS in the wrist anymore.
3. Under lifelike motions options, the micro move doesn't work at all, not that I care too much for this one, just letting you know. I think this one is already issued.

**Request**

I'd love to see back the previous behaviour or an alternative option to choose when calling the UI menu in the VR mode. Right now when calling the menu, this one is aligned with the 3D floor instead of being aligned with the headset camera position... this situation provokes that when the camera is in a "strage" or non-horizontal possition you have to phisically turn or move around looking for the menu to operate. It's very unconfortable and I remember from oldest versions that the menu was aligned with the headset.

Thanks and sorry for the long text...

**Comment by alloystorm** [comment]
There are some issue with the LW shader regarding glow effect, we are trying to fix that. But I can't reproduce what you mentioned about the skin map, it looks all good to me. Is that with a particular model?

**Comment by alloystorm** [comment]
There's a new build 1.4.5.947 for LW, fixed the shader issue.

**Comment by Nyexxi** [comment]
> Hi did you mean 1.4.2 or 1.4.5?

I meant 1.4.5, sorry I wrote that in a wrong way.

> There's a new build 1.4.5.947 for LW, fixed the shader issue.

Nice, thank you... I'll check later.

And what do you think about the menu UI in VR mode? Is it possible to redo it or it is too much trouble? maybe for a new build I guess.

**Comment by Nyexxi** [comment]
Hi, everything in the "Speciphic for LW Desktop and VR Mode" section is completely fixed, well done and many thanks!!

**Comment by alloystorm** [comment]
Yeah we'll need to think about the menu thing. That's not a quick change.
